system/util/dummy_diag_publisher/README.md

```diff
@@ -0,0 +1,33 @@
+# dummy_diag_publisher
+
+## Purpose
+
+This package outputs a dummy diag for debugging and developing.
```

Comment:
IMO: `diag` might be colloquial, so `dummy diagnostic` + _something_ (messages or topics or errors or data?) would be easier to read. In `diagnostic_aggregator/example/README.md`, a sentence `producing dummy diagnostic data` can be found, but I'm not sure which is better.

Reply:
fix "diag" to "diagnostic data"
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/627/commits/48334776cc75e027a44f08084d261f469685032d